js/plugins/googleai/src/gemini.ts

```diff
@@ -451,6 +452,16 @@ export function googleAIModel(
         }
       }
 
+      const strictlyTypedGenerationConfig: GenerationConfig = {
+        candidateCount: request.candidates || undefined,
+        temperature: request.config?.temperature,
+        maxOutputTokens: request.config?.maxOutputTokens,
+        topK: request.config?.topK,
+        topP: request.config?.topP,
+        stopSequences: request.config?.stopSequences,
+        responseMimeType: request.config?.responseMimeType,
```

Comment:
A couple quick thoughts.

- You'll want to add any new fields to the `GeminiConfigSchema` above for proper validation.
- What happens when apiVersion is set to `v1`?

Reply:
> You'll want to add any new fields to the GeminiConfigSchema above for proper validation.

Sure thing, I'll have a look.

>    What happens when apiVersion is set to v1?

If you mean "literally what happens now if you pass through the `responseMimeType` config param?", the Gemini service returns a `400` with a clear error message. This gets thrown all the way out of `genkit`.

```
GoogleGenerativeAIFetchError: [GoogleGenerativeAI Error]: Error fetching from https://generativelanguage.googleapis.com/v1beta/models/gemini-pro:generateContent: [400 Bad Request] Json mode is not enabled for models/gemini-pro
```

Is this adequate or should there be up-front handling? (I considered the `v1` / `v1beta` difference but as there isn't currently any other differentiation in the model implementation, I thought I should go with the flow and not make the distinction.)